nameko/utils/__init__.py

```diff
@@ -134,3 +134,14 @@ def sanitize_url(url):
     parts = parts._replace(netloc='{}:{}@{}'.format(
         parts.username, REDACTED, host_info))
     return parts.geturl()
+
+
+def sanitize_connection_string(s):
```

Comment:
This function should have a name that clearly indicates it can support multiple connection strings, so it isn't mistakenly used instead of `sanitize_url()` for those places that only support a single connection string.

Please rename this function. Also please add a docstring to indicate the supported input types and formats.